draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1104,6 +1104,8 @@ Outputs:
   : A base64 encoded `TransItem` of type `consistency_proof_v2`, whose
     `tree_size_1` MUST match the `first` input. If the `sth` output is omitted,
     then `tree_size_2` MUST match the `second` input.
+    If `first` and `second` are equaly, the returned consistency proof MUST
```

Comment:
I suggest: "s/are equaly,/are equal and correspond to a known STH,/"

If first and second are equal but don't correspond to a known STH, then I think the "first unknown" error would be a more appropriate response than an empty "consistency_path".